### PR TITLE
dotCMS/core#16860 Update saving and saved message in edit page

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss [hunk 1]
```diff
@@ -23,6 +23,15 @@ dot-whats-changed,
...
     justify-content: space-between;
     padding: $basic-padding * 2;
+
+    & > * {
+        display: flex;
+        align-items: center;
+    }
+
+    dot-global-message {
+        display: flex;
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss]: Shouldn't we move this to the host of the `dot-global-message`?

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss [hunk 2]
```diff
@@ -23,6 +23,15 @@ dot-whats-changed,
     display: flex;
     justify-content: space-between;
     padding: $basic-padding * 2;
+
+    & > * {
+        display: flex;
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss]: What is this doing?

File: src/app/view/components/_common/dot-global-message/dot-global-message.component.html [hunk 3]
```diff
@@ -1,2 +1,3 @@
-<span class="dot-global-message__icon {{ message.type }}"></span>
+<dot-icon *ngIf="this.message.type !== 'loading'" name="{{ message.type }}"></dot-icon>
```
Comment (fmontes) [inline src/app/view/components/_common/dot-global-message/dot-global-message.component.html]: you can use `if/else` in the templates.

File: src/app/view/components/_common/dot-global-message/dot-global-message.component.ts [hunk 4]
```diff
@@ -16,20 +16,27 @@ import { DotEvent } from '@models/dot-event/dot-event';
...
+        warning: 'warning'
     };
 
     constructor(private dotEventsService: DotEventsService) {}
 
     ngOnInit() {
         this.dotEventsService
             .listen('dot-global-message')
-            .pipe(filter((event) => !!event.data))
+            .pipe(filter(event => !!event.data))
```
Comment (fmontes) [inline src/app/view/components/_common/dot-global-message/dot-global-message.component.ts]: type?

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss (hunk 2 above)
Comment (hmoreras) [reply src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss]: is making this   https://github.com/dotCMS/core-web/blob/bea62b9a6f5aeeea2cce993604a1a51bff5aa5ce/src/app/portlets/dot-edit-page/content/dot-edit-content.component.html#L12  display flex. to show `<dot-global-message>` correctly

**Pull request merged**